pallets/collator-assignment/src/assignment.rs

```diff
@@ -492,4 +531,5 @@ pub struct ChainNumCollators {
     pub min_collators: u32,
     // This will only be filled if all the other min have been reached
     pub max_collators: u32,
+    pub parathread: bool,
```

Comment:
what does this boolean indicate? whether the chain is a parathread? if so add comment

Reply:
What else could it indicate? xD

Reply:
with our knowledge probably nothing else, but adding a comment does not hurt